e2e/deployers/appset.go

```diff
@@ -18,20 +18,18 @@ func (a ApplicationSet) Deploy(ctx types.Context) error {
 	config := ctx.Config()
 	managementNamespace := ctx.ManagementNamespace()
 
-	drpolicy, err := util.GetDRPolicy(ctx.Env().Hub, config.DRPolicy)
-	if err != nil {
-		return err
-	}
+	// Deploys app on DRCluster(c1)
```

Comment:
What do you mean by `DRCluster(c1)`?

Reply:
Meant first drcluster. Updated to: "// Deploys the application on the first DR cluster (c1)."